Source/OrchBase/P_builder.cpp

```diff
@@ -487,9 +488,10 @@ unsigned P_builder::GenSupervisor(P_task* task)
         sup_inPin_typedefs << " super_InPin_" << sIpin_name << "_props_t;\n\n";
         
         sup_pin_handlers << "   const super_InPin_" << sIpin_name;
-        sup_pin_handlers << "_props_t* sEdgeProperties = ";
+        sup_pin_handlers << "_props_t* sEdgeProperties OS_ATTRIBUTE_UNUSED= ";

```

Comment:
I'm guessing this is compiler warning suppression (which I'm totally fine with here).

You and your sedge/peat bogs.

Reply:
Hey, sEdge isn't me, that is undeniably XML spec ;)

Yep, it is unused warnings suppression on the props and state - not every handler uses them, but they must be available in every handler.